walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/requirements/VerifiedIdRequirement.kt

```diff
@@ -21,7 +21,7 @@ class VerifiedIdRequirement(
     val types: List<String>,
 
     // Constraint that represents how the requirement is fulfilled
-    private val constraint: VerifiedIdConstraint,
+    private val constraint: VerifiedIdConstraint? = null,
```

Comment:
why make this nullable?

Reply:
The only scenario this is null when a VC of specific type is not requested (If the schema is empty in presentation request). Instead of throwing an exception then, this lets the user complete the presentation flow.

Reply:
Made this mandatory since VC Type requested should be specified in presentation request. Fail the presentation if type is missing.